modules/kernel/src/main/scala/trace4cats/kernel/headers/GoogleCloudTraceToHeaders.scala

```diff
@@ -28,26 +28,31 @@ private[trace4cats] object GoogleCloudTraceToHeaders {
   // from https://cloud.google.com/trace/docs/setup
   val headerPattern =
     """(?xi)
-      |([0-9a-f]+) # trace ID
+      |([^\/]+)    # trace ID
```

Comment:
What motivates this change and the one below? Can the trace ID ever been non-hexadecimal or the span ID non-decimal?

Reply:
You will get error in either case. But this way you get more precise error. The regex should try to match only the rough format of the string.